shell/browser/notifications/win/windows_toast_notification.cc

```diff
@@ -83,68 +105,70 @@ WindowsToastNotification::~WindowsToastNotification() {
   // Remove the notification on exit.
   if (toast_notification_) {
     RemoveCallbacks(toast_notification_.Get());
-    Dismiss();
+    // Dismiss();
```

Comment:
?

Reply:
Oh yeah I forget about that. This causes us not to show up in AC. I'm not sure why this was done.